### PR TITLE
feat: ngx.ssl.server_port(): get server port

### PR DESCRIPTION
**Pull request by halfcrazy**
I hereby granted the copyright of the changes in this pull request
to the authors of this lua-resty-core project.

### CODE REVIEW

**Comment by halfcrazy** [comment]
xref https://github.com/openresty/lua-nginx-module/pull/1763

File: lib/ngx/ssl.lua [hunk 1]
```diff
@@ -251,6 +261,25 @@ function _M.raw_server_addr()
...
+    if not r then
+        error("no request found")
+    end
+
+    local rc = ngx_lua_ffi_ssl_server_port(r, uintp, errmsg)
+    if rc == FFI_OK then
+        return uintp[0]
+    end
+
+    if rc == FFI_DECLINED then
```
Comment (spacewander) [inline lib/ngx/ssl.lua:275]: The C code doesn't return `FFI_DECLINED`.

Comment (halfcrazy) [reply lib/ngx/ssl.lua]: update

**Comment by doujiang24** [comment]
@halfcrazy Merged, thanks 👍

**Pull request closed without merging**